api_tests/nodes/views/test_node_list.py

```diff
@@ -1722,6 +1723,140 @@ def test_create_project_errors(
         assert res.status_code == 400
         assert res.json['errors'][0]['detail'] == 'Title cannot exceed 512 characters.'
 
+@pytest.mark.django_db
+class TestNodeLicenseOnCreate:
+
+    @pytest.fixture()
+    def user(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def url(self):
+        return '/{}nodes/'.format(API_BASE)
+
+    @pytest.fixture()
+    def license_name(self):
+        return 'MIT License'
+
+    @pytest.fixture()
+    def node_license(self, license_name):
+        return NodeLicense.objects.filter(name=license_name).first()
+
+    @pytest.fixture()
```

Comment:
FYI, normally permission tests are one of the main things I'm looking for, but anyone can create a node as long as they're logged in, and that test case is covered in creating a node.  There are no special restrictions to adding a license, so none needed.